nxt_editor/__init__.py

```diff
@@ -95,7 +95,7 @@ def _new_qapp():
     return app
 
 
-def launch_editor(paths=None, start_rpc=True):
+def launch_editor(paths=None, start_rpc=True, call_exec=True):
```

Comment:
My approach when I had this need(in the unreal plugin) was to call `_new_app` and `show_new_editor` myself.  Generally I like composing functions together rather than making a swiss army knife function.

Drop it or leave it, won't kill me either way.

Reply:
Yeah I saw what you did, I like it better. Dropping it.